tests/tree_construction.rs

```diff
@@ -1,20 +1,14 @@
-use gosub_engine::testing::tree_construction::fixture_from_filename;
-use lazy_static::lazy_static;
-use std::collections::HashSet;
+use gosub_engine::testing::tree_construction::fixture::{
+    fixture_root_path, read_fixture_from_path,
+};
+use gosub_engine::testing::tree_construction::Harness;
 use test_case::test_case;
 
 const DISABLED_CASES: &[&str] = &[
     // tests18.dat
     "<!doctype html><template><plaintext>a</template>b",
 ];
 
-lazy_static! {
-    static ref DISABLED: HashSet<String> = DISABLED_CASES
-        .iter()
-        .map(|s| s.to_string())
-        .collect::<HashSet<_>>();
-}
```

Comment:
`&[&str].contains(...)` is `O(n)` time, compared to `HashMap::contains(...)`, which is `O(1)` time.  It doesn't matter at all in this case, though, because n is very small (1 right now).

But for a larger set of disabled cases such as the number we had recently, we'd either want the `HashMap` that is being deleted, or we'd want to use [`phf_set!`](https://docs.rs/phf/latest/phf/macro.phf_set.html) (since we're using the phf crate elsewhere).

Reply:
I'm not sure about this. I understand the O(1) hash lookup, but since the hash is generated each run, which is O(n), PLUS the overhead of hashing PLUS the small overhead of searching the hash.

Reply:
`lazy_static!` is macro that allows you to generate once and then never generate again.  So the instantiation only happens once.

Reply:
If a "run" is a run of all of the tests, then, yes, the hash is instantiated once per run.  If a run is the run of an individual test, this would be mistaken.  The `lazy_static!` code only runs once at the start of execution, and then the result is memoized in `ref DISABLED` and is not computed again.